recipes/doxygen/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class DoxygenConan(ConanFile):
+    name = "doxygen"
+    description = "A documentation system for C/C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-2.0"
+    homepage = "http://www.doxygen.nl/"
+    settings = "os_build", "arch_build"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("libiconv/1.15")
+
+    def build_requirements(self):
+        if not tools.which("flex"):
+            self.build_requires("flex/2.6.4@bincrafters/stable")
```

Comment:
Still on review at #1195
```suggestion
            self.build_requires("flex/2.6.4")
```